Python/Product/PythonTools/PythonTools.csproj

```diff
@@ -104,6 +104,9 @@
     <ProjectReference2 Include="$(BuildRoot)Python\Product\ProjectWizards\ProjectWizards.csproj" />
     <ProjectReference2 Include="$(BuildRoot)Python\Product\TestAdapter\TestAdapter.csproj" />
     <ProjectReference2 Include="$(BuildRoot)Python\Product\XamlDesignerSupport\XamlDesignerSupport.csproj" />
+    <Reference Include="Microsoft.VisualStudio.Debugger.Interop.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <EmbedInteropTypes>True</EmbedInteropTypes>
```

Comment:
For the debugger interop assemblies, I think so. It's generally fairly obvious if it isn't going to work.

Check with a Dev15 release whether the `Version=14.0.0.0` part of the name needs to be `Version=$(VSTarget).0.0`